Tasks/Gradle/gradletask.ts

```diff
@@ -5,7 +5,7 @@ import fs = require('fs');
 import path = require('path');
 
 // Lowercased file names are to lessen the likelihood of xplat issues
-import sqCommon = require('sonarqube-common/sonarqube-common');
+import sqCommon = require('./sonarqube-common/sonarqube-common');
```

Comment:
Resolved, done.

I considered leaving them as "sonarqube-server" etc. to match the class names, but that makes a full-name import look like "sonarqube-server.SonarQubeServer" which is just redundant.